src/output-json-metadata.c

```diff
@@ -264,20 +264,32 @@ static OutputInitResult JsonVarsLogInitCtxSub(ConfNode *conf, OutputCtx *parent_
     return result;
 }
 
-void JsonVarsLogRegister (void)
+void JsonMetadataLogRegister (void)
 {
-    OutputRegisterPacketModule(LOGGER_JSON_VARS, MODULE_NAME, "vars-json-log",
-        JsonVarsLogInitCtx, JsonVarsLogger, JsonVarsLogCondition,
-        JsonVarsLogThreadInit, JsonVarsLogThreadDeinit, NULL);
-    OutputRegisterPacketSubModule(LOGGER_JSON_VARS, "eve-log", MODULE_NAME,
-        "eve-log.vars", JsonVarsLogInitCtxSub, JsonVarsLogger,
-        JsonVarsLogCondition, JsonVarsLogThreadInit, JsonVarsLogThreadDeinit,
-        NULL);
+    /* Kept for compatibility. */
```

Comment:
does this comment only belong at the 'vars' registration below?

Reply:
Oops. Yup.